app/src/main/java/org/fossasia/openevent/app/common/app/binding/BindingAdapters.java

```diff
@@ -101,7 +114,39 @@ public static void bindDefaultImage(ImageView imageView, String url, Drawable dr
 
         requestCreator
             .tag(MainActivity.class)
-            .into(imageView);
+            .into(new Target() {
+                @Override
+                public void onBitmapLoaded(Bitmap bitmap, Picasso.LoadedFrom loadedFrom) {
+                    imageView.setImageBitmap(bitmap);
+                    setHeaderColors(bitmap, headerPalette, eventName, eventDate);
+                }
+
+                @Override
+                public void onBitmapFailed(Exception e, Drawable drawable) {
+                    // Nothing doing here
+                }
+
+                @Override
+                public void onPrepareLoad(Drawable drawable) {
+                    // Nothing doing here
+                }
+            });
+    }
+
+    private static void setHeaderColors(Bitmap bitmap, View headerPalette, TextView eventName, TextView eventDate) {
+        Palette.from(bitmap)
+            .generate(palette -> {
+                Palette.Swatch textSwatch = palette.getVibrantSwatch();
```

Comment:
@iamareebjamal 
This is always returning null for me. Please do test on your device if it is working.